elastic_transport/_exceptions.py

```diff
@@ -67,29 +64,14 @@ class SerializationError(TransportError):
 class ConnectionError(TransportError):
     """Error raised by the HTTP connection"""
 
-    def __str__(self) -> str:
-        if self.errors:
-            return f"Connection error caused by: {self.errors[0].__class__.__name__}({self.errors[0]})"
-        return "Connection error"
-
```

Comment:
This `__str__` implementation tries to be helpful but hides the current exception to focus on `errors`, which are past attempts. This comment applies to the other subclasses as well.